packages/collector/test/opentelemetry/test.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const expect = require('chai').expect;
+const path = require('path');
+const semver = require('semver');
+const express = require('express');
+
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../core/test/config');
+const { retry } = require('../../../core/test/test_util');
+const ProcessControls = require('../test_util/ProcessControls');
+const portfinder = require('../test_util/portfinder');
+const globalAgent = require('../globalAgent');
+
+// NOTE: only run on the latest node version
+const mochaSuiteFn =
+  supportedVersion(process.versions.node) && semver.gte(process.versions.node, '18.0.0') ? describe : describe.skip;
+
+mochaSuiteFn('Opentelemetry usage', function () {
```

Comment:
_suggestion:_ I think it might be nice to add a comment that describes the intent of the test. Something like this maybe?

> NOTE: Using @instana/collector and the OpenTelemetry SDK in the same process is *not supported*. Thus, this test does not verify desirable behavior but simply checks what exactly happens when this unsupported setup is used. Both variants (require Instana first/OTel second and vice versa) fail, though they fail in different ways.